<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23467" started-at="2017-12-18T09:08:18Z" finished-at="2017-12-18T09:08:41Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="23467" started-at="2017-12-18T09:08:18Z" finished-at="2017-12-18T09:08:41Z">
      <class name="pfs_System.Create_BQ">
        <test-method status="PASS" signature="setUp()[pri:0, instance:pfs_System.Create_BQ@59fa1d9b]" name="setUp" is-config="true" duration-ms="5" started-at="2017-12-18T12:08:18Z" finished-at="2017-12-18T12:08:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testCal()[pri:0, instance:pfs_System.Create_BQ@59fa1d9b]" name="testCal" duration-ms="23429" started-at="2017-12-18T12:08:18Z" finished-at="2017-12-18T12:08:41Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at pfs_System.Login_Module.tst_contract_login(Login_Module.java:63)
	at pfs_System.Login_Module.testCal(Login_Module.java:117)
	at pfs_System.Create_BQ.testCal(Create_BQ.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCal -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:pfs_System.Create_BQ@59fa1d9b]" name="teardown" is-config="true" duration-ms="1" started-at="2017-12-18T12:08:41Z" finished-at="2017-12-18T12:08:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- pfs_System.Create_BQ -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
